app/controllers/users_controller.rb

```diff
@@ -2,10 +2,13 @@ class UsersController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @users = User.all
   end
 
   def show
+    @user = User.find(params[:id])
+    @follow = User.where(id: current_user.id)
+    @posts = @user.posts.order(created_at: :desc)
+    @user_likes = @user.likes.where(post_id: @posts.pluck(:id)).index_by(&:post_id)
```

Comment:
These variables are in load_users and load_social_actions functions. :c

Reply:
so?

Reply:
![image](https://github.com/BrightCoders-Institute/s10a1-proyecto-final-team-03-s10a1-proyecto-final/assets/73453092/cc7e2d1a-d689-4b71-a573-eb1ce968c9a3)

These functionar are called in  `show`, therefore us rendundant. :c

Reply:
Ohhh jajajaj I didn't see at all. Thanks